### ISSUE TITLE
Warning when running grunt PostCSS task after initial run

### ISSUE THREAD

**Issue by allenmoore** [jonathantneal/mdcss#5]
I've added mdcss to the PostCSS grunt tasks for an upcoming project. When running grunt, after running grunt to generate the initial style guide, a warning is thrown in the PostCSS grunt task. This is a copy of the warning:

```
Fatal error: Error: ENOENT, unlink '/Users/username/path/to/style-guide/examples.js',
Error: ENOENT, unlink '/Users/username/path/to/style-guide/prism.js',
Error: ENOENT, unlink '/Users/username/path/to/style-guide/script.js',
Error: ENOENT, unlink '/Users/username/path/to/style-guide/mdcss-logo.png',
Error: ENOENT, unlink '/Users/username/path/to/style-guide/style.css'
```

I have custom options in place for mdcss that change the directory name, title, and uses a custom logo. In order to determine if the warning is due to the custom options, I remove the options and used the mdcss defaults. I then deleted the previously generated style guide, and ran grunt twice. On the second run, the same warning appears as shown above.

I receive this warning no OS X El Capitan 10.11.1.

**Comment by zrothauser** [comment]
I'm getting the same problem, with the following settings:

```
require('mdcss')({
    theme: require( 'mdcss-theme-github' ),
    destination: 'test/visual/styleguide'
})
```

**Comment by jonathantneal** [comment]
Are you still having this issue? There has not been an `unlink` method used in mdcss, and any `fs` calls are either synchronous within try / catches or integrated with Promises.

**Comment by allenmoore** [comment]
I am still having this issue. Yesterday, I integrated mdcss into a new project. I tested with the defaults, plus changing the default directory to a custom location. In every instance, this error occurred.

Is there a potential conflict with another grunt module? I can provide a package.json example of that would help.

**Comment by jonathantneal** [comment]
Yes, please do provide whatever information should allow me to replicate the issue. If it’s a conflict, then it would be valuable to understand the conflict and either mention it in the documentation or file an issue with the conflicting project.

**Comment by jonathantneal** [comment]
Some systems have issue overwriting files without first clearing the directory. This has been resolved in 1.5.0. Thanks, @allenmoore!
